commons/passive/fitbit/fitbit_sleep_stages.avsc

```diff
@@ -0,0 +1,25 @@
+{
+  "namespace": "org.radarcns.passive.fitbit",
+  "type": "record",
+  "name": "FitbitSleepStages",
```

Comment:
The fields of this schema seem identical to the ones in the `FitbitSleepClassic` schema. Is that intentional? Could we just use one schema `FitbitSleep`, and send the data to two topics (`fitbit_sleep_classic` and `fitbit_sleep_stages`)?

Reply:
Yes I agree. There is no need for two schemas